samples/dialogs/src/main/java/com/badoo/ribs/samples/dialogs/rib/Dialogs.kt

```diff
@@ -0,0 +1,14 @@
+package com.badoo.ribs.samples.dialogs.rib
+
+import com.badoo.ribs.android.dialog.CanProvideDialogLauncher
+import com.badoo.ribs.core.Rib
+import com.badoo.ribs.core.customisation.RibCustomisation
+
+interface Dialogs : Rib {
```

Comment:
(Use plugin to rename all classes automatically)

```suggestion
interface DialogsExample : Rib {
```